### PR TITLE
Documintation for SetLogLevel

### PR DESCRIPTION
**Pull request by ArtiomDivak**
Wrote some documintation of how and where setloglevel is implemanted.

### CODE REVIEW

File: doc/docs/api/description.md [hunk 1]
```diff
@@ -194,6 +198,10 @@ Object path: `/org/containers/hirte/node/$name`
 
     `Reload()` may be invoked to reload all unit files.
 
+  * `SetLogLevel(in s loglevel)`
+
+    Set log level for hirte-agent by invoking the internal hirte agebt API.
```
Comment (dougsland) [inline doc/docs/api/description.md]: typo: :s/agebt/agent/g

**Review by dougsland — COMMENTED** [review]
the PR title has a typo: 

:s/Documintation/Documentation/g

Other than that looks good to me.

File: doc/docs/api/description.md [hunk 2]
```diff
@@ -347,6 +355,10 @@ This is the main interface that the node implements and that is used by the mana
   * `Reload()`
 
     `Reload` causes systemd on the agent to reload all unit files.
+  
+  * `SetLogLevel()`
```
Comment (engelmi) [inline doc/docs/api/description.md]:
```suggestion
  * `SetLogLevel(in s log_level)`
```
Doesn't it require the log level as input parameter?

File: doc/docs/api/description.md [hunk 3]
```diff
@@ -194,6 +198,10 @@ Object path: `/org/containers/hirte/node/$name`
 
     `Reload()` may be invoked to reload all unit files.
 
+  * `SetLogLevel(in s loglevel)`
```
Comment (engelmi) [inline doc/docs/api/description.md]:
```suggestion
  * `SetLogLevel(in s log_level)`
```
I think for the parameters we use `snake_case`. Could you please change that for the others as well? (I know we are not 100% consistent here - so if you want to change the other `camelCases` to `snake_case` that would be awesome :))

File: doc/docs/api/description.md [hunk 4]
```diff
@@ -41,6 +41,10 @@ Note that all properties also come with change events, so you can easily track w
 
     Disables the collection of metrics on all agents.
 
+  * `SetLogLevel(in s loglevel)`
+
+    Set new log level for hirte manager.
```
Comment (engelmi) [inline doc/docs/api/description.md]: Could you mention that this change is not being persisted and is only effective as long as hirte/-agent is not restarted?

File: doc/docs/api/description.md [hunk 5]
```diff
@@ -41,6 +41,10 @@ Note that all properties also come with change events, so you can easily track w
 
     Disables the collection of metrics on all agents.
 
+  * `SetLogLevel(in s log_level)`
+
+    Set new log level for hirte manager. This change is persistent as long as hirte not restarted.
```
Comment (mwperina) [inline doc/docs/api/description.md]: I'd change to 

*Set the new log level ...*

And also shouldn't we use the word *controller* instead of *manager*?

**Review by mwperina — COMMENTED** [review]
Please also fix typos in commit subject and message

**Comment by mwperina** [comment]
Could you please split commits for unrelated changes? Right now I see 2 commits:

1. Changes around set log level
2. Changes around parameter names

And please don't forget about typos in commit's subject and message

**Comment by mwperina** [comment]
Please perform the correct rebase on top of latest change. Other than that it looks good to me.

File: doc/docs/api/description.md [hunk 6]
```diff
@@ -351,6 +359,10 @@ This is the main interface that the node implements and that is used by the mana
   * `Reload()`
 
     `Reload` causes systemd on the agent to reload all unit files.
+  
+  * `SetLogLevel(in s log_level)`
+
+    Set the new log level for hirte-agent node. This change is persistent as long as hirte-agrnt not restarted.
```
Comment (mwperina) [inline doc/docs/api/description.md]: hirte-agrnt -> hirte-agent

**Review by mwperina — CHANGES_REQUESTED** [review]

**Review by engelmi — APPROVED** [review]
LGTM

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**